cumulusci/cli/cci.py

```diff
@@ -65,13 +65,16 @@ def timestamp_file():
         yield f
 
 
-def get_latest_version():
+def get_latest_final_version():
     """ return the latest version of cumulusci in pypi, be defensive """
     # use the pypi json api https://wiki.python.org/moin/PyPIJSON
     res = requests.get("https://pypi.org/pypi/cumulusci/json", timeout=5).json()
     with timestamp_file() as f:
         f.write(bytes(time.time()))
-    return pkg_resources.parse_version(res["info"]["version"])
+    versions = [pkg_resources.parse_version(v) for v in res["releases"].keys()]
```

Comment:
we already have a LOT of cci releases and more coming, is there any benefit to only parsing and checking the last N? or is this all so fast we dn't care

Reply:
Since they seem to be sorted alphabetically and we can have many betas, I'm not sure that's safe.